src/ui/zcl_abapgit_gui_page_tag.clas.abap

```diff
@@ -271,7 +271,7 @@ CLASS zcl_abapgit_gui_page_tag IMPLEMENTATION.
 
     ro_html->add( render_text_input( iv_name  = 'sha1'
                                      iv_label = 'SHA1'
-                                     iv_value = mo_repo_online->get_sha1_remote( ) ) ).
+                                     iv_value = mo_repo_online->get_remote_branch_sha1( ) ) ).
```

Comment:
this is a method rename, it can be done in a separate PR

Reply:
is this critical ? If you insist I can do but this is really a couple of line in all code